10._Positioning_Charter.md

```diff
@@ -0,0 +1,87 @@
+
+### Evaluate
+When evaluating SLSA compared to other standards/frameworks/regulations, we will aim to answer one or more of the following questions: 
+- Should SLSA increase/decrease scope?
+- How does SLSA work with other frameworks? 
+- Is there overlap in SLSA with other frameworks?
+- Are there deficiencies or out of scope items in SLSA with relation to other frameworks?
+- Are there use cases/personas to address that were not considered in SLSA?
+
+Community Members may leverage one or more of the following mediums to perform evaluations:  
+- Google Doc/Spreadsheet
+- GitHub Issue
+- Whiteboarding Application (i.e. Mural, etc.) 
+
+After evaluations are performed, this SIG will take a position on its impacts for SLSA and organizations.
+
+### Educate/Evangelize
+Purpose of education is to highlight the benefits that can be gained by using SLSA. Additionally, positioning SIG members may highlight the current/future work, and where the community is needing help.  When educating the open source community, global industry, and standards/regulatory bodies on SLSA, we will aim to perform one or more of the following:
+
+- Educate organizations on how SLSA compares to other frameworks, regulations, or standards
+- Educate organizations of how to apply SLSA to their use cases
+- Educate organizations on out-of-scope items for SLSA
+- Educate standards/regulatory bodies on how SLSA fits or fills in deficiencies in standards/regulations, by referencing the comparative evaluations performed.
+- Educate other open source communities on how to leverage SLSA to improve code quality/security via self-elected workstream or via an existing initiative (i.e. OpenSSF WG - Critical Software)
+
+Community Members may leverage one or more of the following mediums:  
+- SLSA Blog (see Contribution Guidelines for more info)
+- SLSA Slack Channels (see Contributor Covenant Code of Conduct)
+- SLSA/OpenSSF/Community Meetings
+- OpenSSF/Linux Foundation Slack/Social Media accounts (See Respective Contributions Guidelines)
+- Personal Social Media accounts (i.e. LinkedIn, Twitter, etc.)
+- Conferences (i.e. Presentations, Lightning talks, Hallway talks, etc.)
+
+### Encourage
+By performing the previously stated missions, we should encourage participation from the broader community/industry to obtain diverse perspectives, which will in turn improve SLSA. By encouraging participation with diverse perspectives, it may help us with:
+
+- Identifying new use cases 
+- Identifying new personas
+- Identifying opportunities for SLSA expansion
+- Identifying opportunities for shifting SLSA strategy/scope
+- Enabling additional OpenSSF participation
+- Improving security of our open source communities/industry
+
+## Operating Goals
+In addition to the Collaboration/Workstream process documented in [SLSA SIGs document](https://docs.google.com/document/d/1L1gEJMBIvE0IbpFi23FOUByDYlItSYPPJmKdhvJQYsg/edit#heading=h.1hce59kd4nn0), the following are our short term/long term goals:
```

Comment:
nit: can we make this charter stand-alone without linking to original proposal? It's not clear to me what, if anything, one is supposed to read in that doc related to this section.

Same below.